backends/dpdk/dpdkArch.cpp

```diff
@@ -1230,7 +1228,15 @@ const IR::P4Table* SplitP4TableCommon::create_group_table(const IR::P4Table* tbl
 const IR::Node* SplitActionSelectorTable::postorder(IR::P4Table* tbl) {
     bool isConstructedInPlace = false;
     bool isAsInstanceShared = false;
-    auto instance = Helpers::getExternInstanceFromProperty(tbl, "psa_implementation",
+    cstring implementation = "psa_implementation";
```

Comment:
implementation shall be initialized to empty string if "psa_implementation" is not default.
If "psa.." is default then below if-check shall be removed.

Reply:
Thanks! Updated.